lib/src/pdf_page_proxy.dart

```diff
@@ -28,22 +28,22 @@ class PDFPageProxy {
 
   PDFPageProxy._withJsInternal(this._jsInternal);
 
-  int get pageNumber => _jsInternal['pageNumber'] as int;
+  int? get pageNumber => _jsInternal['pageNumber'] as int?;
 
-  int get rotate => _jsInternal['rotate'] as int;
+  int? get rotate => _jsInternal['rotate'] as int?;
```

Comment:
Used to pull in page rotation from the pdf itself, not related to the rotations we store in the graph. Would be good to test on some PDFs that have rotated pages before we import